build-tests/sparo-real-repo-test/etc/checkout-to.txt

```diff
@@ -0,0 +1,35 @@
+Running "sparo checkout --to sparo-output-test":
```

Comment:
If the CLI is changing, we need to update the corresponding website docs:
https://tiktok.github.io/sparo/pages/commands/sparo_checkout/

Reply:
updated